### ISSUE TITLE
Update checker

### ISSUE THREAD

**Issue by LemADEC** [sheenrox82/TheMistsOfRioVII#9]
The update checker is called 3 times whenever players login, doing 3 times the HTTP request, it's a bit slow...

When update is found, the link in chat isn't clickable.

Only OP players needs to know about new updated version. Normal players have no need for the spam since anyway, it's a server, they can't do much about it :). Doing the check only when player is OP would save the lag when they connect to the server.

**Comment by sheenrox82** [comment]
... Well for one, it's not called three times at all. And two, every player needs to know about the updated version, and three, it's not supposed to be clickable.

Not a bug.

Won't fix.

**Comment by LemADEC** [comment]
https://github.com/sheenrox82/TheMistsOfRioVII/blob/a66beddbcbb66192e6c4ed9081a464543e7747dd/java/sheenrox82/riovII/src/event/Events.java#L66
https://github.com/sheenrox82/TheMistsOfRioVII/blob/a66beddbcbb66192e6c4ed9081a464543e7747dd/java/sheenrox82/riovII/src/event/Events.java#L76
and
https://github.com/sheenrox82/TheMistsOfRioVII/blob/a66beddbcbb66192e6c4ed9081a464543e7747dd/java/sheenrox82/riovII/src/event/Events.java#L86

that makes 3 times calling the same function.

**Comment by LemADEC** [comment]
As for the context, a small modpack typically has 60+ mods. Try to imagine for a second what happens when they all try to check for update whenever client connects to a server. At best they've got lag, at worse they've got mobs/players killing them while they're still checking for updates.
A proper implementation is to check for update in a separate thread (do not hold the game), only once and without impacting others (typically when the game is starting at Init phases).
A proper display is only for OP & SSP games, ideally in the main menu.
Actually, this would make a great library for many mods :).

**Comment by sheenrox82** [comment]
Hmm, could've sworn I only called once. Will roll out a patch in a minute to "fix" (?) this "issue". 

As for the other parts, my points stand. All users need to see it, and they aren't supposed to be clickable.

**Comment by LemADEC** [comment]
Guess I'll have to patch it so users can actually see the server MOTD then

**Comment by sheenrox82** [comment]
Not my fault you have 60+ mods in your server. >.>

**Comment by sheenrox82** [comment]
Hmm, perhaps, I can add an option for OPs to see only.

Will be disabled by default. Eh. Probably doesn't matter now. For the most part this issue was 'fixed'.

**Comment by LemADEC** [comment]
An disabled by default, "OP only" option sounds good to me.
